src/module.cpp

```diff
@@ -1925,6 +1935,39 @@ bool Module::writeDevStub(const char *fn) {
     return true;
 }
 
+bool Module::writeCPPStub(const char *outFileName) { return writeCPPStub(this, outFileName); }
+
+bool Module::writeCPPStub(Module *module, const char *outFileName) {
+    // Get a file descriptor corresponding to where we want the output to
+    // go.  If we open it, it'll be closed by the llvm::raw_fd_ostream
+    // destructor.
+    //
+    int fd;
+    int flags = O_CREAT | O_WRONLY | O_TRUNC;
+
+    if (!outFileName) {
+        return false;
+    } else if (!strcmp("-", outFileName)) {
+        fd = 1;
+    } else {
+#ifdef ISPC_HOST_IS_WINDOWS
+        flags |= O_BINARY;
```

Comment:
It's not binary data, so probably we don't need it. As I understand it will affect the end of lines.